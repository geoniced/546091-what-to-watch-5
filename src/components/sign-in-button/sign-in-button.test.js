import React from "react";
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import SignInButton from "./sign-in-button";

it(`renders SignInButton component`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <SignInButton />
        </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
