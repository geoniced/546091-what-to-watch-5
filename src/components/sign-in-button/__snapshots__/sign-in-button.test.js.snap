// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SignInButton component 1`] = `
<a
  className="user-block__link"
  href="/login"
  onClick={[Function]}
>
  Sign in
</a>
`;
